MetaCell/salk-interactive-atlas
# #185 feat: Update meshes and cordmap

Closes #185 

Implemented solution: 
- Updates meshes
- Updates cordmap

How to test this PR: 
- Create an experiment with the following files
[V2b_PKD2L1_Cord1_Key.csv](https://github.com/MetaCell/salk-interactive-atlas/files/9461089/V2b_PKD2L1_Cord1_Key.csv)
[V2b_PKD2L1_Cord1_Data.csv](https://github.com/MetaCell/salk-interactive-atlas/files/9461090/V2b_PKD2L1_Cord1_Data.csv)
- Confirm that no ring structure is displayed and the cells in the 3D viewer match with the cells in the 2D viewer.

### Sanity checks:

### Breaking changes (select one):

### Possible deployment updates issues (select one):

### Test coverage (select one):

### Documentation (select one):
